<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>2D</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>911</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Novo ponto:</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ponto_x">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ponto_y">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>ND</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Vetor:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>221</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Classe:</string>
    </property>
   </widget>
   <widget class="QLabel" name="classe_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="distancia_group">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>120</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Dist√¢ncia:</string>
    </property>
    <widget class="QRadioButton" name="hamming_radio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Hamming</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="euclides_radio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Euclides</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="classificador_group">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Classificador: </string>
    </property>
    <widget class="QRadioButton" name="neighbor_radio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nearest neighbor</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vizinhos:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="numero_vizinhos">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>430</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Classificar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="entrada_menu">
    <property name="title">
     <string>Entrada</string>
    </property>
    <addaction name="carregar_arquivo"/>
    <addaction name="gerar_espiral"/>
   </widget>
   <addaction name="entrada_menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="carregar_arquivo">
   <property name="text">
    <string>Carregar arquivo</string>
   </property>
  </action>
  <action name="gerar_espiral">
   <property name="text">
    <string>Gerar espiral</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>gerar_espiral</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>gerarEspiral()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>classificar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>gerarEspiral()</slot>
  <slot>classificar()</slot>
  <slot>definirClassificador()</slot>
  <slot>definirDistancia()</slot>
 </slots>
</ui>
